squad_client/core/models.py

```diff
@@ -216,6 +226,10 @@ class BuildMetadata(SquadObject):
     pass
 
 
+class BuildStatus(SquadObject):
```

Comment:
Just a thougt/question: creating a BuildStatus obj makes sense. Would it also make sense to make it nested class inside build, since we won't ever populate it individually without going thru Build. Not major but just a thought.

Reply:
Right now it's a standalone endpoint in squad, so IMHO it should stay.